tests/cromwell_options.json

```
@@ -2,6 +2,7 @@
     "final_workflow_outputs_dir": "test-output",
     "use_relative_output_paths": true,
     "default_runtime_attributes": {
-        "docker_user": "$EUID"
+        "docker_user": "$EUID",
+        "memory": "5G"
```

```suggestion
        "memory": "2 GiB"
```
To match the default from the wdl spec